src/main/webapp/mentors.js

```diff
@@ -48,15 +50,75 @@ function createMentorElement(name, description) {
 
   const mentorName = document.createElement('h5');
   mentorName.className = 'card-title text-primary';
-  mentorName.innerHTML = name;
+  mentorName.innerHTML = mentor.name;
   mentorCardBody.appendChild(mentorName);
 
+  mentorCardBody.appendChild(createGitHubLink(mentor.gitHubID));
+
   const mentorDescription = document.createElement('p');
-  mentorDescription.innerHTML = description;
+  mentorDescription.innerHTML = mentor.description;
   mentorCardBody.appendChild(mentorDescription);
 
+  mentorCardBody.appendChild(createTagRow(mentor.interestTags));
+
+  mentorCardBody.appendChild( createProjectsRow(mentor.projectIDs));
+
   mentorCard.appendChild(mentorCardBody);
   mentorContainer.appendChild(mentorCard);
 
   return mentorContainer;
 }
+
+/**
+ * Creates a github profile button element from username.
+ * @param {String} gitHubID
+ * @return {HTMLElement} userGithubButton
+ */
+function createGitHubLink(gitHubID) {
+  let gitHubBaseUrl = 'https://github.com/';
+  const gitLink = gitHubBaseUrl.concat(gitHubID);
+  let userGithubButton = document.createElement('a');
+  userGithubButton.innerText = 'GitHub Profile';
+  userGithubButton.className = 'btn btn-primary';
+  userGithubButton.role = 'button';
+  userGithubButton.href = gitLink;
+  return userGithubButton;
+}
+
+/**
+ * Creates a block of interest tags.
+ * @param {String[]} interestTags
+ * @return {HTMLElement} tagDiv
+ */
+function createTagRow(interestTags) {
```

Comment:
nit: call this `createTagDiv` because `createTagRow` makes me think that this is creating a `<tr>` object.

Reply:
Thank you!